GUI application.py

```diff
@@ -1333,7 +1341,7 @@ def __init__(self, parent, parameters, picoscope, Keyboard_feedback, siglent, fe
         self.cyclesLabel = QLabel()
         self.outputLabel = QLabel()
 
-        self.freqLabel.setText('Frequency (kHz)')
+        self.freqLabel.setText('Frequency (mHz)')
```

Comment:
Is mHz here milli (1e-3) Herz? Or do you mean mega (1e6) Hertz i.e. MHz? The difference being that the M is capitalized in mega Hertz.

Reply:
367c4b7 changes the label to mega hertz.